main.py

```diff
@@ -75,5 +95,6 @@ def main():
     parser.add_argument('-xgb', action='store_true', help='Run with XGBoost Model')
     parser.add_argument('-nn', action='store_true', help='Run with Neural Network Model')
     parser.add_argument('-A', action='store_true', help='Run all Models')
+    parser.add_argument('--odds', default='fanduel', help='Sportsbook to fetch from. (fanduel, draftkings, betmgm, pointsbet, caesars, wynn, bet_rivers_ny')
```

Comment:
Can we change this to just `'-odds'` to match format.

Reply:
done